### ISSUE TITLE
Action Facade\Ignition\Http\Controllers\ExecuteSolutionController not defined

### ISSUE THREAD

**Issue by cashweaver** [monicahq/monica#3860]
**Describe the bug**
I see a "Action Facade\Ignition\Http\Controllers\ExecuteSolutionController not defined" error when attempting to save a change to the "Time of the day reminders should be sent" in /settings.

**Screenshots**
![1587237501_1218_18042020_1073x1433](https://user-images.githubusercontent.com/56796112/79669835-67bec880-8173-11ea-9356-724ab11d804b.png)

**Which version are you using:**
Docker, monica/monica:fpm, Version: 2.17.0.

**Additional context**
This error occurs with a fresh container. The steps taken before attempting to save the time are:

1. Stop all monica docker containers (`docker-compose down`)
1. Destroy all monica docker containers (`docker system prune --volumes`)
1. Start the database container, let it complete it's set up, then stop it (`docker-compose up db`)
1. Start the Monica, database, and web containers as detached (`docker-compose up -d monica web db`)
1. Create first user account (/register)
1. Run setup:production (`docker-compose exec -T monica php artisan setup:production --force`)
1. Remove log file with inappropriate file permissions, see #575 (`docker-compose exec -T monica rm /var/www/monica/storage/logs/laravel-2020-04-18.log`)
1. Visit the settings page (/settings) and attempt to update the "Time of the day reminders should be sent".

**Comment by cashweaver** [comment]
docker-compose.yaml:
```yaml
version: '3.7'

services:
  monica:
    depends_on:
      - db
    env_file: 
      - ./monica/.env 
      - ./monica/.env-secrets
    image: monicahq/monicahq:fpm
    restart: always
    volumes: 
      - vol-data:/var/www/monica/storage 
      - vol-www:/var/www/monica

  cron:
    command: cron.sh
    depends_on: 
      - db
    env_file: 
      - ./monica/.env
      - ./monica/.env-secrets
    image: monicahq/monicahq:fpm
    restart: always
    volumes: 
      - vol-data:/var/www/monica/storage
      - vol-www:/var/www/monica:ro

  queue:
    command: queue.sh
    depends_on: 
      - db
    env_file: 
      - ./monica/.env
      - ./monica/.env-secrets
    image: monicahq/monicahq:fpm
    restart: always
    volumes: 
      - vol-data:/var/www/monica/storage
      - vol-www:/var/www/monica:ro

  db:
    env_file:
      - ./db/.env
      - ./db/.env-secrets
    image: mysql:5.7
    restart: always
    volumes: 
      - vol-mysql:/var/lib/mysql

  web:
    build: ./web
    depends_on:
      - monica
    env_file:
      - ./web/.env
    restart: always
    volumes:
      - vol-data:/var/www/monica/storage:ro
      - vol-www:/var/www/monica:ro

volumes:
  vol-data: 
    name: vol-data
  vol-www: 
    name: vol-www
  vol-mysql:
    name: vol-mysql

networks:
  default:
    external:
      name: nginx-proxy
```

Other files can be seen in my [self-hosted repo](https://github.com/cashweaver/self-hosted/tree/7c8ec8824f283f20e4807b8c82862f2c06c56019/monica).

**Comment by cashweaver** [comment]
Oddly, I am able to successfully save (it survives browser refresh) the Timezone without error.

**Comment by cashweaver** [comment]
The same error also occurs when I visit `/settings/storage`.

**Comment by cashweaver** [comment]
I ran the following artisan commands based on [advice in a similar discussion](https://github.com/laravel/framework/issues/30298#issuecomment-574065073) and tried to save the time again.

1. `docker-compose exec -T monica php artisan cache:clear`
1. `docker-compose exec -T monica php artisan route:clear`

Instead of saving successfully, I saw a nginx 502 error page upon clicking "Update Preferences". Then, upon refreshing and confirming that I wanted to resubmit the form, I got a different error: BadMethodCallException Call to undefined method App\Models\Account\Account::hasLimitations().

![Screenshot 2020-04-19 at 6 27 14 PM](https://user-images.githubusercontent.com/56796112/79705853-f3b41b80-826b-11ea-8b7b-9561360e8a8c.png)

**Comment by cashweaver** [comment]
I restarted the instances (`docker-compose down && docker-compose up -d`) and noticed that the [original error](https://github.com/monicahq/monica/issues/3860#issue-602533864) has returned.

**Comment by cashweaver** [comment]
This issue appears to be a duplicate of #3748.

**Comment by cashweaver** [comment]
@asbiin When should we expect to see that fix (#3792) included in the Docker images?

**Comment by cashweaver** [comment]
The current latest docker image hosted on dockerhub does not contain #3792. Instead of waiting for that image to be updated, I did the following:

1. Add this repo as a submodule in my self-hosted repo
`git submodule add https://github.com/monicahq/monica.git`
1. Checkout the #3792 commit
`git checkout 597b97fa68122141fb9c9308af6b2c62bbdacbe6`
1. Built a new local image based on the `fpm`
`docker build -t monicalocal .`
1. [Updated my docker-compose file to include a reference to the new image](https://github.com/cashweaver/self-hosted/commit/3ad6053e415d13f797917c1570f82f9093f6da47)
1. Restarted monica
`docker-compose down && docker-compose up -d`
